src/lights/PointLightShadow.js

```diff
@@ -64,7 +67,7 @@ class PointLightShadow extends LightShadow {
 		const camera = this.camera;
 		const shadowMatrix = this.matrix;
 
-		const far = light.distance || camera.far;
+		const far = light.distance != null ? light.distance : DEF_DISTANCE;
```

Comment:
`null` isn't a valid value for light.distance so this should be `light.distance !== 0 ? light.distance : DEF_DISTANCE` or `light.distance || DEF_DISTANCE` since 0 indicates no limit,